### PR TITLE
Adding support for city api

### PR DESCRIPTION
**Pull request by radhikakhetan**

### CODE REVIEW

File: resources/cities.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+        "city": "Ahemdabad",
+        "state": "Gujarat"
+    },
+    {
+        "city": "Anand",
+        "state": "Gujarat"
+    }
+]
```
Comment (kamaljoshi) [inline resources/cities.json]: newline

File: routes/index.js [hunk 2]
```diff
@@ -15,7 +15,11 @@ module.exports = {
 				items = data[resource];
 				return returnSingle(items, itemArg, res);
 			}
-			items = search(data[resource], req, res, true);
+			if (resource === 'cities') {
+				items = search(data[resource], req, res, false);
```
Comment (kamaljoshi) [inline routes/index.js]: Why do you have to do this? Resources are auto picked.

Comment (radhikakhetan) [reply routes/index.js]: In all other resources type matching is exact. I want the matching to be partial. Hence this condition

File: resources/cities.json (hunk 1 above)
Comment (radhikakhetan) [reply resources/cities.json]: done

File: routes/helper.js [hunk 3]
```diff
@@ -28,18 +28,24 @@ module.exports = {
...
         _.each(filters, function(value, key) {
             if (exactMatch) {
                 filteredItems = _.filter(filteredItems, function(item) {
                     return (getSanitizedValue(item[key]) === getSanitizedValue(value));
                 });
-            } else {
-                filteredItems = _.filter(filteredItems, function(item) {
-                    return getSanitizedValue(item[key]).indexOf(getSanitizedValue(value)) > -1;
-                });
+            } else {
```
Comment (kamaljoshi) [inline routes/helper.js]: Don't do second level of nesting. Just use else if.

Comment (radhikakhetan) [reply routes/helper.js]: done

**Comment by kamaljoshi** [comment]
Can you rename your commit message to be something more descriptive. Look at the previous commits to get an idea.

**Comment by radhikakhetan** [comment]
> Can you rename your commit message to be something more descriptive. Look at the previous commits to get an idea.

Done

**Pull request merged**